tests/fixtures/test_alfasim_runner.py

```diff
@@ -96,3 +115,20 @@ def test_alfasim_runner_from_case_description(
     results = alfasim_runner.run()
     assert alfasim_runner.alfacase_file.is_file()
     assert len(results.list_profiles()) == 2
+
+
+def test_alfasim_runner_invalid_results(
+    alfasim_runner: AlfasimRunnerFixture,
+    fake_alfasim: None,
+    empty_alfacase: Path,
+    fake_alfasim_return_code,
+) -> None:
+    from alfasim_sdk import convert_alfacase_to_description
+
+    case_description = convert_alfacase_to_description(Path(empty_alfacase))
+
+    alfasim_runner.load_base_from_case_description(case_description)
```

Comment:
Is there coverage issues?
Why not just

```suggestion
) -> None:
    alfasim_runner.load_base_from_alfacase(empty_alfacase)
```

Reply:
They are not. I just had copied the previous test and didn't notice I could have used directly the `load_base_from_alfacase` function. Thanks for the suggestion.